plugins/inputs/system/PROCESSES_README.md

```diff
@@ -39,6 +40,7 @@ Linux  FreeBSD  Darwin  meaning
   R       R       R     running
   S       S       S     sleeping
   Z       Z       Z     zombie
+  X       X       X     dead
```

Comment:
This switch is only over the first character though, and as I read it X cannot be the first char on these other platforms.